test/safe_math/safe_math.js

```diff
@@ -0,0 +1,122 @@
+const BigNumber = require('bn.js');
+const SafeMathTest = artifacts.require('SafeMathTest');
+const utils = require('../test_lib/utils');
+
+let safeMathInstance;
+contract('SafeMath', function () {
+	const MAX_UINT = new BigNumber(2).pow(new BigNumber(256)).sub(new BigNumber(1));
+	
+	beforeEach(async function () {
+		
+		safeMathInstance = await SafeMathTest.new();
```

Comment:
Why "safeMathInstance" and not simply "safeMath"? A quick search through openst-contracts (and mosaic-contracts) suggests this is a departure from our typical practice—for any particular reason (i.e., it is meant to indicate something special in this case)?